popSimManu.tex

```diff
@@ -620,10 +631,10 @@ \section*{Discussion}
 are generally similar.
 In these comparisons, \fastsimcoal and \dadi perform almost identically, which is expected because they fit the same models to the same summaries of the data, differing only in how they calculate model expectations and optimize parameters.
 
-All methods for inferring demographic history have strengths and weaknesses (as recently reviewed by \citet{beichman2018review}.)
+All methods for inferring demographic history have strengths and weaknesses (as recently reviewed by \citep{beichman2018review}.)
```

Comment:
I think this should be `\citet{ }`, unless you want to do it like `\citep[][as recently reviewed by]{beichman2018review}` (or maybe `\citep[as recently reviewed by][]{beichman2018review}`, I forget).

Reply:
ah yea I was having trouble getting some of the citations to compile and thought for a second that might be the problem